package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vuejs-datepicker",
-  "version": "0.2.10",
+  "version": "0.2.11",
```

Comment:
Can you return this to 0.2.10 and once I merge the PR I'll update this with an `npm version patch` command